packages/tools/client-debugger/client-debugger/src/messaging/Messages.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Message structure expected for window event listeners used by the Fluid Client Debugger.
+ *
+ * @public
+ */
+export interface IDebuggerMessage<TData = unknown> {
+	/**
+	 * The source of the event.
```

Comment:
I added a remarks block with some extra details, but I would like to avoid documenting anything here explicitly in terms of the devtools extension, since that is just one consumer.